qr-code/src/test/java/com/adyen/checkout/qrcode/DefaultQRCodeDelegateTest.kt

```diff
@@ -198,7 +198,7 @@ internal class DefaultQRCodeDelegateTest(
                     Activity(),
                 )
 
-                assertEquals(awaitItem(), QrCodeComponentViewType.QR_CODE)
+                assertEquals(awaitItem(), QrCodeComponentViewType.SIMPLE_QR_CODE)
```

Comment:
Can you add a similar test to this but passing `PAY_NOW` and asserting `FULL_QR_CODE`?